src/components/TeamProfileCards/index.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import Translate from '@docusaurus/Translate';
+import Link from '@docusaurus/Link';
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+function WebsiteLink({to, children}) {
+  return (
+    <Link to={to}>
+      {children || (
+        <Translate id="team.profile.websiteLinkLabel">website</Translate>
+      )}
+    </Link>
+  );
+}
+
+function TeamProfileCard({className, name, role, children, avatar, links}) {
+  return (
+    <div className={className}>
+      <div className="card card--full-height">
+        <div className="card__header">
+          <div className="avatar avatar--vertical">
+            <img
+              className="avatar__photo avatar__photo--xl"
+              src={avatar}
+              alt={`${name}'s avatar`}
+            />
+            <div className="avatar__intro">
+              <h3 className="avatar__name">{name}</h3>
+              {role && (
+                <h4 className="avatar__role">{role}</h4>
+              )}
+            </div>
+          </div>
+        </div>
+        <div className="card__body">{children}</div>
+        <div className="card__footer">
+          <div className="button-group button-group--block">
+            {links && links.map(({title, url}, index) => (
+              <a key={index} className="button button--secondary" href={url}>
+                {title}
+              </a>
+            ))}
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+function TeamProfileCardCol(props) {
+  return (
+    <TeamProfileCard {...props} className={'col col--6 margin-bottom--lg'} />
+  );
+}
+
+export function CoreTeamRow() {
+  return (
+    <div className="row">
+      <TeamProfileCardCol
+        name="Ricardo Lüders"
+        role="Project Leader &amp; Engineer Leader"
+        avatar="https://github.com/rluders.png"
+        links={[
+          {title: "Github", url: "https://github.com/rluders"},
+          {title: "LinkedIn", url: "https://www.linkedin.com/in/ricardoluders"},
+          {title: "Twitter", url: "https://twitter.com/rluders_"},
+        ]}>
+        <Translate id="team.profile.Ricardo Lüders.body">
+          Holistic Software Engineer (the universe will fix the bug 🔮), Project Leader and Founder of the Sumé LMS, 
+          that love technology, open-source, education, cats 🐱, video game 🎮, and music 🎼.
+        </Translate>
+      </TeamProfileCardCol>
+      <TeamProfileCardCol
+        name="Ariane Rocha"
+        role="Data Engineer Leader"
```

Comment:
Data Engineer at Magrathea Labs, Leader of Data Sumé, and one of the founders of the project Sumé.
Passionate about technology, open-source, games, books, comics, podcasts, and cats, much more for cats actually.